### PR TITLE
Rework readme

### PR DESCRIPTION
**Pull request by hakunanatasha**
Updates the readme to do the following:

(1) Reflect the new policies of the hackathon
(2) Contributing guide now highlights how to make a PR etc. details for dataloader creation
(3) upload hub instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,34 +21,25 @@ There are two broad categories of biomedical datasets:
...
-## Get started
-
-### Pre-Requisites
+*Optional* If you are looking for the official guides for contributing to the `datasets` library from Huggingface for a [shared dataset](https://huggingface.co/docs/datasets/share_dataset.html) or to [add a dataset](https://huggingface.co/docs/datasets/add_dataset.html), you can find them in the links attached. Our guide follows closely from these, with adaptations to suit this intitiative.
 
-Please do the following before getting started: 
+## Get started!
 
-- [Make](https://huggingface.co/join) an account on 🤗's Hub and [login](https://huggingface.co/login). **Choose a good password, as you'll need to authenticate your credentials**. 
+Please make a github account prior to implementing a dataset; you can follow instructions to install git [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
```
Comment (sunnnymskang) [inline README.md:38]:
```suggestion
Please [make a github account](https://github.com/join) prior to implementing a dataset; you can follow instructions to install git [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). 
```

File: README.md [hunk 2]
```diff
@@ -19,34 +21,25 @@ There are two broad categories of biomedical datasets:
...
-    - click the "Request to join this org" button in the upper right corner.
+You will also need at Python 3.6+. If you are installing python, we recommend downloading [anaconda](https://docs.anaconda.com/anaconda/install/index.html) to curate a python environment with necessary packages. 
 
-- Make a github account; you can follow instructions to install git [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). 
-
-*Optional* <br>
-- You will also likely need at Python 3.6+. If you are installing python, we recommend downloading [anaconda](https://docs.anaconda.com/anaconda/install/index.html) to curate a python environment with necessary packages. 
+All commands in the guide provided are done through terminal access.
 
 ### 1. Choose a dataset to implement
```
Comment (sunnnymskang) [inline README.md:45]: @hakunanatasha @jason-fries At this moment, I can't comment on the dataset board: https://github.com/orgs/bigscience-workshop/projects/6/views/1  - I'll leave a thread on Slack regarding it

File: README.md [hunk 3]
```diff
@@ -75,62 +68,50 @@ In your new issue, please provide the following details:
...
+2. [Translation](templates/template_translation.py)
+3. [Part-of-speech (POS)](templates/template_pos.py)
+4. [Question-answering (QA)](templates/template_qa.py)
+5. [Natural language inference (NLI)](templates/template_nli.py)
+6. [Normalization](templates/template_normalization.py)
+7. [Sentence classification](templates/template_sentcls.py)
+8. [Fact verification](templates/template_factver.py)
+9. [Semantic similarity](templates/template_semsim.py)
+10. [Topic classification](templates/template_topiccls.py)
+11. [Co-reference](templates/template_coref.py)
```
Comment (sunnnymskang) [inline README.md:96]: @hakunanatasha Here, are we intending to direct them to template script under templates/<> for each tasks (which doesn't exist at the moment)  OR direct them to schemas? (under schema)

**Review by sunnnymskang — CHANGES_REQUESTED** [review]
Please see my comments above

File: README.md (hunk 3 above)
Comment (galtay) [reply README.md:96]: I'm starting to think we don't need a template for each task? What if we just had one template and then made it a goal to implement one full example for each task type in the examples directory (and made sure there was at least one local files example)? I feel like there is a lot of boiler plate in the templates and that a lot of the differences are covered in the schemas directory and README, 
* https://github.com/bigscience-workshop/biomedical/tree/master/schemas
* https://github.com/bigscience-workshop/biomedical/blob/master/task_schemas.md

Comment (hakunanatasha) [reply README.md:96]: Here's what I'm thinking:

(1) A markdown page discussing and describing all schemas
(2) Each task example pointing to the schema and 1 implemented example
(3) one UNIVERSAL template that walks through what each of the core functions of the dataloader is

File: README.md (hunk 1 above)
Comment (hakunanatasha) [reply README.md:38]: nice thanks

File: README.md (hunk 2 above)
Comment (hakunanatasha) [reply README.md:45]: python 3.6 is a requirement for `datasets` from what i gather; i'll keep the phrasing but have a comment on this later.

Comment (hakunanatasha) [reply README.md:45]: also i don't think you can comment on the project board as that's owned by bigscience, you can comment on issues, however.

File: README.md (hunk 3 above)
Comment (galtay) [reply README.md:96]: yes, that sounds like a nice format :+1:

File: README.md [hunk 4]
```diff
@@ -55,121 +54,87 @@ There are two options to choose a dataset to implement; you can choose either op
...
-<!---
-@NATASHA Make DDI script
+## FAQs
 
-[Example with BRAT format annotation](examples/ddi.py)
--->
+#### What if my dataset does not have a public license?
 
-### 5. Commit your changes
+We understand that some biomedical datasets require external licensing. To respect the agreement of the license, we recommend implementing a dataloader script that works if the user has a locally downloaded file. You can find an example [here](examples/cellfinder.py) and follow the local [template](templates/template_local.py).
```
Comment (galtay) [inline README.md:100]:
```suggestion
We understand that some biomedical datasets require external licensing. To respect the agreement of the license, we recommend implementing a dataloader script that works if the user has a locally downloaded file. You can find an example [here](examples/cellfinder.py) and follow the local/private dataset specific instructions in  [template](templates/template.py).
```

**Review by sunnnymskang — APPROVED** [review]

**Pull request merged**